src/Serilog.Sinks.Console/project.json

```diff
@@ -0,0 +1,60 @@
+{
+  "version": "2.0.0-beta-*",
+  "description": "The console sink for Seriolog - Simple .NET logging with fully-structured events",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "logging", "semantic", "structured", "console" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-nuget.png",
+  "frameworks": {
+    "net40": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Microsoft.Bcl.Async": "1.0.168",
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "net45": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "dnx451": {
+      "compilationOptions": {
+        "keyFile": "../../assets/Serilog.snk"
+      },
+      "frameworkAssemblies": {
+        "System.Configuration": ""
+      },
+      "dependencies": {
+        "Serilog": "2.0.0-beta-*"
+      }
+    },
+    "dotnet5.4": {
+      "dependencies": {
+        "Microsoft.CSharp": "4.0.1-beta-23516",
+        "Serilog": "2.0.0-beta-*",
+        "System.Collections.Concurrent": "4.0.11-beta-23516",
+        "System.Console": "4.0.0-beta-23516",
+        "System.Diagnostics.Process": "4.1.0-beta-23516",
+        "System.Diagnostics.TraceSource": "4.0.0-beta-23516",
+        "System.IO": "4.0.11-beta-23516",
+        "System.IO.FileSystem": "4.0.1-beta-23516",
```

Comment:
I'm not sure we need this? The dependencies should probably be shaved down for this package?